packages/shared/ReactFeatureFlags.js

```diff
@@ -164,6 +164,9 @@ export const enableCustomElementPropertySupport = __NEXT_MAJOR__;
 // request for certain browsers.
 export const enableFilterEmptyStringAttributesDOM = __NEXT_MAJOR__;
 
+// Disabled caching behavior of `react/cache` in client runtimes.
+export const disableClientCache = false;
```

Comment:
Any harm to enabling this in experimental now and setting it to `__NEXT_MAJOR__`?